pynxtools/dataconverter/readers/ellips/reader.py

```diff
@@ -361,7 +363,17 @@ def populate_header_dict_with_datasets(file_paths, is_mock=False):
         header[f"data_collection/NAME_spectrum[{spectrum_type}_spectrum]"] = (
             whole_data[spectrum_type].to_numpy()[0:counts[0]].astype("float64"))
 
+        def write_scan_axis(name: str, values: list, units: str):
+            base_path = f"Env_Conditions/PARAMETER[{name}]"
+            header[f"{base_path}/values"] = values
+            header[f"{base_path}/values/@units"] = units
+            header[f"{base_path}/number_of_parameters"] = len(values)
+            header[f"{base_path}/number_of_parameters/@units"] = ""
```

Comment:
Should the number of parameters have units? Is there a mistake in the template generation or NXDL here?

Reply:
This I discussed with @sanbrock already. I was also arguing for not having a unit specified on this field but @sanbrock s argument was to set it to `NX_UNITLESS` to not accidentally set a wrong number.